worlds/bk_sudoku/docs/setup_en.md

```diff
@@ -0,0 +1,22 @@
+# BK Sudoku Setup Guide
+
+## Required Software
+- [Bk Sudoku](https://github.com/Jarno458/sudoku)
+- [.Net 6](https://docs.microsoft.com/en-us/dotnet/core/install/windows?tabs=net60)
+
+## General Concept
+
+This is a universal client, it can connect to any multiworld slot, and lets you play some sudoku to unlock random hints.
+
+## Installation Procedures
+
+Download latest release on [BK Sudoku Releases](https://github.com/Jarno458/sudoku/releases). Download and extract the Bk_Sudoku.zip file.
+
+## Joining a MultiWorld Game
+
+1. Run Bk_Sudoku.exe
+2. Enter the name of the slot you wish to connect to
+3. Enter the server url & port
+4. Press connect
+5. Choose difficulty
+6. Try to solve the Sudoku
```

Comment:
For this list of instructions, I suggest punctuating them as sentences (you know, adding periods, etc). That's just my own personal idea of what looks nice, though.